requirements.txt

```diff
@@ -7,13 +7,15 @@ future==0.16.0
 gunicorn==19.9.0
 hanzidentifier==1.0.2
 hiredis==0.2.0
+kdtree
 lxml==4.6.2
 mapbox-vector-tile==1.2.0
 ModestMaps==1.4.7
 protobuf==3.4.0
 psycopg2==2.7.3.2
 pyclipper==1.0.6
 pycountry==17.9.23
+pyshp==2.3.0
```

Comment:
apparently my system thought i needed these, i didnt check to see if that was the case. i can revert